docs/decisions/0003-name-affirmation-signals.rst

```diff
@@ -0,0 +1,38 @@
+3. Using Signals for Name Affirmation
+-------------------------------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+A `VerifiedName` object has a `status` field that should correspond to the status of the associated IDV or proctored exam attempt.
```

Comment:
Can you please expand out `IDV` to `Identity Verification`?

Reply:
Fixed this, and also added the set of options to the doc.